crane-cloud/frontend
# Add user profile on admin side

# Description

Add user profile on admin side
## Type of change


## Trello Ticket ID
https://trello.com/c/B98tdAsp

## How Can This Been Tested?
Log into the admin side and check out the users list. Click a user and check out their profile

## Checklist:


# Screenshots
![Screenshot from 2022-08-03 22-39-16](https://user-images.githubusercontent.com/32802973/182696750-c4a381b3-acff-41e6-ae89-35f1298905f7.png)
![Screenshot from 2022-08-02 11-16-52](https://user-images.githubusercontent.com/32802973/182699855-897ffa2d-6c7e-4c56-84c1-c89734565582.png)

![Screenshot from 2022-08-03 17-52-35](https://user-images.githubusercontent.com/32802973/182696763-824c67b8-5133-4378-bc7a-24df73b06cc5.png)
![Screenshot from 2022-08-03 17-35-53](https://user-images.githubusercontent.com/32802973/182696767-ac8e1635-1029-47f3-939f-7fdf21548707.png)
![Screenshot from 2022-08-03 17-35-43](https://user-images.githubusercontent.com/32802973/182696773-9ab93fa5-1eef-4b77-aecf-515476a77717.png)
![Screenshot from 2022-08-03 17-29-54](https://user-images.githubusercontent.com/32802973/182696778-29b0a807-64d5-485a-a384-c2b371b5ff92.png)
![Screenshot from 2022-08-03 17-03-42](https://user-images.githubusercontent.com/32802973/182696779-36842664-3c3b-45c2-a6b5-2c19b01f1102.png)
